modules/gapi/test/common/gapi_parsers_tests_common.hpp

```diff
@@ -233,145 +234,163 @@ class ParserYoloTest
         return mat;
     }
 
-    void parseYoloRef(const cv::Mat& in_mat,
-                      const cv::Size& in_sz,
+    void parseYoloRef(const cv::Mat&  in_yolo_result,
+                      const cv::Size& in_size,
                       const float& confidence_threshold,
                       const float& nms_threshold,
                       const int& num_classes,
                       const cv::gapi::core::YoloAnchors& anchors,
                       std::vector<cv::Rect>& out_boxes,
                       std::vector<int>& out_labels)
     {
-        auto detections = parse(in_mat.ptr<float>(),
-                                confidence_threshold,
-                                in_sz.height,
-                                in_sz.width,
-                                num_classes,
-                                anchors);
+        YoloParams params;
+        constexpr auto side = 13;
+        constexpr auto side_square = side * side;
+        const auto output = in_yolo_result.ptr<float>();
```

Comment:
As I can see, output is never used after it is stored in the class field `m_out`. Is it possible to store `in_yolo_result.ptr<float>()` right there?

Reply:
Fixed.